storage/spec/helpers.rb

```diff
@@ -0,0 +1,43 @@
+require "google/apis/cloudkms_v1"
```

Comment:
Change name of file to `spec_helpers.rb`.

Reply:
Done.